ecc/bls12-377/fr/poseidon2/poseidon2_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Consensys Software Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Code generated by consensys/gnark-crypto DO NOT EDIT
+
+package poseidon2
+
+import (
+	"testing"
+
+	"github.com/consensys/gnark-crypto/ecc/bls12-377/fr"
+)
+
+func TestExternalMatrix(t *testing.T) {
```

Comment:
Is there some test vectors out there as for poseidon?https://extgit.iaik.tugraz.at/krypto/hadeshash/-/blob/master/code/test_vectors.txt